packages/gatsby-recipes/src/providers/gatsby/__snapshots__/plugin.test.js.snap

```diff
@@ -18,7 +18,7 @@ Object {
 module.exports = {
   /* Your site config here */
   plugins: [],
-}
+};
```

Comment:
What's up with the semi's in this file?

Reply:
probably cause we run prettier on the gatsby-config.js source after we add a plugin — so without a .prettierrc around for tests, it adds semis by default